test/apple/fastfile.test.ts

```diff
@@ -46,4 +44,490 @@ describe('fastlane', () => {
       });
     });
   });
+
+  describe('#findIOSPlatform', () => {
+    describe('platform block detection', () => {
+      const variations: {
+        name: string;
+        content: string;
+        expected: { index: number; length: number };
+      }[] = [
+        {
+          name: 'no platform',
+          content: 'lane :test do\nend',
```

Comment:
`l`: This is a bit hard to read with the inline `\n`. It would be great if we could split this up over multiple lines instead.